test/e2e/set/scyllacluster/datainserter.go

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) 2021 ScyllaDB
+
+package scyllacluster
+
+import (
+	"context"
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/gocql/gocql"
+	"github.com/scylladb/gocqlx/v2"
+	"github.com/scylladb/gocqlx/v2/table"
+	scyllav1 "github.com/scylladb/scylla-operator/pkg/api/scylla/v1"
+	"k8s.io/apimachinery/pkg/util/rand"
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
+)
+
+const nRows = 10
+
+type DataInserter struct {
+	client            corev1client.CoreV1Interface
+	session           *gocqlx.Session
+	keyspace          string
+	table             *table.Table
+	data              []*TestData
+	replicationFactor int
+}
+
+type TestData struct {
+	Id   int
+	Data string
+}
+
+func NewDataInserter(ctx context.Context, client corev1client.CoreV1Interface, sc *scyllav1.ScyllaCluster, replicationFactor int) (*DataInserter, error) {
+	if replicationFactor < 1 {
+		return nil, fmt.Errorf("replication factor can't be set to less than 1")
+	}
+
+	keyspace := strings.Replace(string(sc.UID), "-", "", -1)
```

Comment:
guess I was ambiguous before - I though UID of the DataInserter, not ScyllaCluster - iow. generate a random sting (and save it), like with `k8s.io/apimachinery/pkg/util/uuid.NewUUID()`

Reply:
this way you can have 2 independent DataInserters on the same ScyllaCluster

Reply:
In this case we'd need to restore `Update` or some method to overwrite expected data as combining `NewDataInserter` and `Insert` would result in creating a new table in a different keyspace instead of overwriting data as before.

Reply:
why do we need to overwrite preexisting data? I though we agreed not to do the update case, so in the most complex case of scaling this would become:
 - scale==1
 - write to Keyspace foo
 - read from keyspace foo
 - scale==2
 - write to keyspace bar
 - read key space bar
 - possibly scale back to 1 replica and read with lowered quorum

Reply:
I guess one other option would be to alter keyspace replication factor after scaling up

Reply:
In this case explicitly setting RF to 1 after scaling up should suffice. I changed `keyspace` name to one generated with a `UID` per `DataInserter` instance and removed `IF NOT EXISTS` in keyspace and table creation since it wouldn't be necessary now. Same with `Clear`.